ops-bedrock/docker-compose.yml

```diff
@@ -157,3 +171,16 @@ services:
 #      - "9090:8080"
 #    volumes:
 #      - op_log:/op_log:ro
+
+  #in no way should this be running in production
```

Comment:
So does this KMS instance serve the key that was derived from the previous mnemonic? I see the seed below, but it's not obvious how those keys were derived

Reply:
Great, that's what I assumed.  Could you add a comment maybe describing that?  Bonus points if a few shell commands to reproduce could be included.

Reply:
I added comments. I didn't find shell commands to generate the private key from mnemonic. I used the `ethers` package to calculate the private key.